build.sbt

```diff
@@ -1,10 +1,21 @@
 import Dependencies.Library
 
+lazy val commonSettings = Seq(
+  publishMavenStyle := true,
+  publishTo := {
+    val prefix = if (isSnapshot.value) "snapshots" else "releases"
+    Some(s"Models$prefix" at s"s3://library-artifacts-$prefix.hubofallthings.com")
```

Comment:
Models should be replaced with "HAT Library Artifacts" as this is what it was previously

Reply:
done